src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -10440,3 +10440,114 @@ ByteBuffer QuestPOIWrapper::BuildQueryData() const
 
     return tempBuffer;
 }
+
+void ObjectMgr::SendServerMail(Player* player, uint32 id, uint32 reqLevel, uint32 reqPlayTime, uint32 rewardMoneyA, uint32 rewardMoneyH, uint32 rewardItemA, uint32 rewardItemCountA, uint32 rewardItemH, uint32 rewardItemCountH, std::string subject, std::string body, uint8 active) const
```

Comment:
void ObjectMgr::SendServerMail(Player* player, const ServerMail& data)

Reply:
Only if i want the workflow to fail.

Reply:
@acidmanifesto I don't see how it would fail passing this much arguments is strange when you have a struct with the exact variables, but I guess that's your choice. 
At least pass the string as a const reference or use std::string_view to avoid the unnecessary copies.
Also it might be a good idea to use a vector for the items, what if they want to send multiple items?